pkg/resources/globalrolebinding/globarolebinding.go

```diff
@@ -36,6 +40,27 @@ func (grbv *globalRoleBindingValidator) Admit(response *webhook.Response, reques
 	// Pull the global role to get the rules
 	globalRole, err := grbv.globalRoles.Get(newGRB.GlobalRoleName)
 	if err != nil {
+		if errors.IsNotFound(err) {
```

Comment:
Can we do an opposite check here and then return if it isn't then the fallthrough is what exists inside this if check

Reply:
Yep, I'll change